Chirp/Pages/ChatsPage/ChatsViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ChatsViewController.swift
+//  Chirp
+//
+//  Created by Ady on 6/7/23.
+//
+
+import UIKit
+
+class ChatsViewController: UIViewController {
+    
+    var chatsTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.allowsSelection = true
+        tableView.backgroundColor = .systemBackground
+        return tableView
+    }()
+    
+    var searchController: UISearchController = {
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchBar.placeholder = "Search"
+        return searchController
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupNavigationBarTitle()
+        setupSearchBar()
+        setupViews()
+        setupConstraints()
+    }
+    
+    private func setupNavigationBarTitle() {
+        title = "Chats"
+        navigationController?.navigationBar.prefersLargeTitles = true
+        // TODO: Check if this is not needed
+        navigationItem.hidesBackButton = true
```

Comment:
You are correct. This is not needed.
We will set this view controller as root view controller so there will be no back button to hide.